client-lib/src/main/java/com/ibm/ws/repository/strategies/writeable/BaseStrategy.java

```diff
@@ -93,10 +94,11 @@ protected State calculateTargetState(RepositoryResourceImpl matchingResource) {
 
     /**
      * Delegates to the resource.findMatchingResource, see {@link RepositoryResourceImpl#findMatchingResource()}
+     * 
+     * @throws RepositoryResourceNoConnectionException
```

Comment:
Would it be better to add all exceptions to the javadoc? Or none, I'd be happy with either